node/src/blockchain/blockchain_interface/blockchain_interface_web3/rpc_helpers_web3.rs

```diff
@@ -164,7 +164,18 @@ mod tests {
             )
             .unwrap();
 
-        assert_eq!(result, U256::from(65_535));
+        assert_eq!(result, U256::from(3_735_928_559_u64));
```

Comment:
If you could represent this in hexadecimal, it'd be easier to match up with the Arrange part of the test.